scripts/install.sh

```diff
@@ -80,6 +86,8 @@ printf "${LOG_START}Starting installation...${LOG_END}"
 
 printf "Network: $NETWORK\n"
 
+printf "${LOG_WARNING_START}Make sure you have at least ${RECOMMENDED_ACCOUNTS_NUMBER} accounts${LOG_WARNING_END}"
```

Comment:
During unlocking 
Maybe we could be more precise and say:
"Make sure you have at least ${RECOMMENDED_ACCOUNTS_NUMBER} ethereum accounts"
?
We later (during unlocking) call them "ethereum accounts".

Reply:
Sure [abcb8a2](https://github.com/keep-network/keep-core/pull/3151/commits/abcb8a2f2e1d044fb2a35360c5c808909c087cc0)